_data/maintainers.yml

```diff
@@ -1165,3 +1165,7 @@ zookal:
   label: Zookal
   name: zookal
   url: http://zookal.com/
+btheunissen:
```

Comment:
@BTheunissen the maintainer name and the variant name are usually expected to match. Would it be ok to label this as maintained by `shaped-ai` since it lives in that github namespace?

Reply:
Thanks @pnadolny13 just accepted those changes and fixed the maintainer to Shaped